### PR TITLE
Attributions

### PR DESCRIPTION
**Pull request by dai-chen**
*Issue #, if available:*

*Description of changes:* Add attributions doc and referenced in README.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/attributions.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+3. For complex queries like JOIN or Multi-query, there was no concept of a query plan for planning and optimizing.
+4. ES-SQL uses a handwritten SQL parser Druid which is not extensible, so many queries that not supported or has semantical error can still pass the parsing but ended up throwing runtime error.
+
+
+
+## What major improvements we made?
+
+Apart from the problems we identified earlier, we made significant improvement in terms of functionality and reliability:
+
+1. *Integration Test*: we migrated all integrate tests to standard Elasticsearch IT framework which spin up in-memory cluster for testing. Now all test cases treat plugin code as blackbox and verify functionality from externally.
```
Comment (abbashus) [inline docs/attributions.md]: Should we start the sentences with uppercase letters?

Comment (abbashus) [inline docs/attributions.md]: np: spin --> spins

Comment (dai-chen) [reply docs/attributions.md]: Sure, will correct. Thanks!

Comment (dai-chen) [reply docs/attributions.md]: > Should we start the sentences with uppercase letters?

You mean "we" right?

Comment (abbashus) [reply docs/attributions.md]: Yes, `we `, `logical` and `a new`.

Comment (dai-chen) [reply docs/attributions.md]: Got it. I missed that. Thanks for proofread. :)

**Pull request merged**